Report.md

```diff
@@ -72,7 +73,28 @@ Stakeholders refer to individuals/entities that have an interest in the developm
 
 Design Thinking Approach
 ========================
-*3. Describe your approach from a Design Thinking principles perspective. Your Design Thinking approach to address the problem with artefacts*
+
+## **Approach - Design Thinking**
+Design thinking is a non-linear, iterative process which seeks the understanding of users to challenge assumptions, redefine problems and create innovative solutions to prototype and test. This customer-centric approach was undertaken to address the improvements requested by the Major Travel Company for their Call Management Centre (CMC). The design thinking approach undertook consists of the execution of five phases, (1) empathize, (2) define, (3) ideate, (4) prototype and (5) test and by addressing these phases we derived customer-centric innovation. 
+
+### Empathise
+Phase one consisted of empathising with stakeholders, this represents anyone who is impacted by the project being developed with a particular focus extended on the end customer. Empathy is a crucial step of achieving a human-centred design process and was utilised to gain insight into users and their needs without introducing any pre imposed bias. When we create solutions, we get better solutions when we think of those who they will affect. To document these perspectives, empathy maps were utilised to collaboratively visualise and share our understanding of stakeholders needs and aid future decision making. This stage allowed us to gain a deeper understanding of the perspective of the users involved and how to shape our solution. 
+
+### Define
+Phase two of defining allowed us to synthesise the knowledge obtained in the empathise stage and define the core problems to solve. Here the problem statements were directly derived from creating point of view (POV) statements for all the stakeholders identified. This staged allowed us as a team to consolidate assumptions and agree on features to be developed to solve the problem given. 
```

Comment:
"Phase two of defining"
Phase two of design thinking _is_ defining?
Also "This stage" not staged

Reply:
Also "This stage" not staged - totally didn't catch that  will change 
Phase two of design thinking is defining? Yea it is the wording if off will change it